tests/damage_rheology_latent_heat/statistics

```diff
@@ -0,0 +1,23 @@
+# 1: Time step number
+# 2: Time (seconds)
+# 3: Time step size (seconds)
+# 4: Number of mesh cells
+# 5: Number of Stokes degrees of freedom
+# 6: Number of temperature degrees of freedom
+# 7: Number of degrees of freedom for all compositions
+# 8: Iterations for temperature solver
+# 9: Iterations for composition solver 1
+# 10: Iterations for Stokes solver
+# 11: Velocity iterations in Stokes preconditioner
+# 12: Schur complement iterations in Stokes preconditioner
+# 13: Minimal temperature (K)
+# 14: Average temperature (K)
+# 15: Maximal temperature (K)
+# 16: Minimal value for composition olivine_grain_size
+# 17: Maximal value for composition olivine_grain_size
+# 18: Global mass for composition olivine_grain_size
+0 0.000000000000e+00 0.000000000000e+00 320 3303 1449 1449  0  0 25 26 26 9.99000000e+02 9.99500000e+02 1.00000000e+03 8.00000000e-05 8.00000000e-05 4.00000000e+06 
+1 2.916757647458e+14 2.916757647458e+14 320 3303 1449 1449 34 22 20 21 21 9.99017651e+02 1.00006385e+03 1.00484664e+03 8.00000000e-05 1.33641962e-03 9.12582750e+06 
+2 5.833396885589e+14 2.916639238131e+14 320 3303 1449 1449 25 17 18 19 19 9.99028974e+02 1.00063223e+03 1.00987049e+03 8.00000000e-05 1.82413197e-03 1.15262122e+07 
+3 8.750015227292e+14 2.916618341703e+14 320 3303 1449 1449 25 16 16 17 17 9.99038070e+02 1.00121282e+03 1.01454952e+03 8.00000000e-05 2.07596557e-03 1.30863246e+07 
+4 1.000000000000e+15 1.249984772708e+14 320 3303 1449 1449 15 10 14 15 15 9.99041607e+02 1.00146560e+03 1.01643772e+03 8.00000000e-05 2.15063808e-03 1.36339991e+07 
```

Comment:
The original values (from your old branch) are 9.99041621e+02 1.00146708e+03 1.01643824e+03 (for min/max/mass of temperature). So this is very similar to the original test results. 
This is surprising to me, as the latent_heat_enthalpy test was very different from the old branch. So maybe for the latent_heat_enthalpy the resolution is just too coarse.